runelite-client/src/main/java/net/runelite/client/plugins/runecraft/AbyssRifts.java

```diff
@@ -64,18 +64,14 @@
 	@Getter
 	private final int itemId;
 
-	private static final Map<Integer, AbyssRifts> rifts;
+	private static final Map<Integer, AbyssRifts> rifts = new HashMap<>();
```

Comment:
Why did you change this?

Reply:
No clue, I didn't even touch anything in the Abyss stuff o.o